tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -838,6 +838,44 @@ exports[`resource_inflight.w > stderr 1`] = `
 "
 `;
 
+exports[`resource_init.w > stderr 1`] = `
+"error: Multiple initializers defined in class R
+  --> ../../../examples/tests/invalid/resource_init.w:3:3
+  |
+3 |   init() {}
+  |   ^^^^^^^^^ Multiple initializers defined in class R
+
+
+error: Multiple inflight initializers defined in class R
+  --> ../../../examples/tests/invalid/resource_init.w:6:3
+  |
+6 |   inflight init() {}
+  |   ^^^^^^^^^^^^^^^^^^ Multiple inflight initializers defined in class R
+
+
+error: Unexpected 'ERROR'.
+  --> ../../../examples/tests/invalid/resource_init.w:9:12
+  |
+9 |   inflight init(x: num) {}
+  |            ^^^^^ Unexpected 'ERROR'.
```

Comment:
Can we improve this error message (or create an issue if it seems out of scope of this PR)

Reply:
fixed by adding inflight init parameters as valid grammar and error reporting in _parser.rs_